internal/api/attestationconfigapi/fetcher.go

```diff
@@ -57,31 +56,17 @@ func newFetcherWithClientAndVerifier(client apifetcher.HTTPClient, cosignVerifie
 
 // FetchAzureSEVSNPVersionList fetches the version list information from the config API.
 func (f *fetcher) FetchAzureSEVSNPVersionList(ctx context.Context, attestation AzureSEVSNPVersionList) (AzureSEVSNPVersionList, error) {
+	// TODO (derpsteb): Replace with FetchAndVerify once we move to v2 of the config API.
```

Comment:
Why is this only possible for v2 of the config API?
If this is something we want to do, we could simply start uploading signatures for the list object every time we update it.
This would not be a breaking change for the v1 API, since nothing existing changes (we just add an extra optional file to process)